extensions.html

```diff
@@ -95,8 +95,7 @@
           // document unless you know what you're doing. If in doubt ask your friendly neighbourhood
           // Team Contact.
           wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/83482/status",
-
-          xref: ["DOM", "HTML", "INFRA", "gamepad"],
+          xref: ["HTML", "DOM", "PERMISSIONS-POLICY", "HR-TIME", "INFRA", "gamepad"],
```

Comment:
```suggestion
          xref: ["HTML", "DOM", "PERMISSIONS-POLICY", "HR-TIME", "INFRA", "gamepad","webxr-gamepads-module"],
```
This should solve the ReSpec/xref issue

Reply:
I think that's not sufficient to solve the issue. The failure is in index.html in a part of the document not modified by the PR, so I suspect it was a change in respec that caused this failure.

@marcoscaceres said it looks like a respec bug. The xref sees &quot; instead of double-quote characters so it fails to match against the webxr-gamepads-module reference.